File: two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int,int> lookup;
        vector<int> fin;
        for(int i=0;i!=nums.size();i++){
            lookup.insert(pair<int,int>(nums[i],i));
        }
        for(int i=0;i!=nums.size();i++){
            auto x=lookup.find(target-nums[i]);
            if(x!=lookup.end() && x->second!=i){
                fin.push_back(i);
                fin.push_back(x->second);
                break;
            }
        }
        return fin;
    }
};

File: two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        output=[]
        for i in range(0,len(nums)):
            for j in range(0,len(nums)):
                if i!= j:
                    if nums[i]+ nums[j]==target:
                        output.append(i)
                        output.append(j)
                        return output
            
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        